locales/ko.yaml

```diff
@@ -0,0 +1,582 @@
+cant_take_more_orders: ⛔ 죄송합니다! 진행 중인 주문이 있을 때에는 다른 주문을 수락할 수 없습니다.
+seller_released: 💸 판매자가 이미 사토시의 잠금을 해제했습니다. 결제가 완료될 때까지 잠시만 기다려주세요.
+your_purchase_is_completed: |
+  🪙 당신의 비트코인 주문이 성공적으로 완료되었습니다. @${sellerUsername}은 당신의 법정화폐 결제 입금을 확인하였고, 봇을 통해 당신의 인보이스에 결제를 완료했습니다. 사운드 머니 경험을 즐겨주세요!
+
+  ⚡️🍊⚡️
+problem_getting_price: 환율 데이터 접근에 문제가 발생했습니다. 잠시 뒤에 다시 시도해주세요. 문제가 계속된다면 관리자에게 문의해주세요.
+update_user_setting: ${field} 필드를 ${newState}로 업데이트하였습니다.
+lightning_address_disabled: 라이트닝 주소를 비활성화하였습니다.
+invalid_range_with_amount: |
+  범위는 유동 비율로만 가능합니다.
+
+  범위를 지정하거나 액수를 지정해주세요. 둘 다 사용하는 것은 안 됩니다.
+too_many_pending_orders: 동시에 등록할 수 있는 최대 주문 갯수까지 등록하였습니다.
+name: 이름
+group: 그룹
+channel: 채널
+solver: 분쟁해결자
+published: 등록됨
+created: 생성됨
+yes: 네
+no: 아니오
+wizard_add_invoice_init: |
+  🤖 ${currency} ${fiatAmount}에 해당되는 ${satsAmount} 사토시의 인보이스를 보내주세요. 그래야 법정화폐 입금 처리가 가능합니다.
+
+  ${expirationTime} 분 내에 인보이스를 보내주시지 않으면, 해당 거래는 취소됩니다.
+wizard_add_invoice_exit: |
+  설정 메뉴에서 나오셨습니다. 이제 일반 명령어를 적으실 수 있으며, 주문 ID와 인보이스를 명시하여 /setinvoice 명령어를 통해 새로운 인보이스를 주문에 추가할 수 있습니다. ${amount} 사토시가 명시된 인보이스를 보내주시거나, 액수를 명시하지 않은 인보이스를 보내주실 수 있습니다:
+
+  `/setinvoice ${orderId} `\\<액수가 명시되거나 명시되지 않은 라이트닝 인보이스\\>
+wizard_community_enter_name: 당신의 커뮤니티 이름을 정해주세요.
+wizard_community_too_long_name: 커뮤니티 이름은 ${length} 글자 내에서 지어야 합니다.
+wizard_community_enter_currency: 커뮤니티 안에서 사용자들이 사용할 법정화폐의 코드(들)를 입력해주세요. 하나 이상일 경우, 스페이스로 구분해 주셔야 합니다.
+wizard_community_enter_group: |
+  그룹을 위한 ID나 이름을 정해주세요. 봇과 당신은 그룹의 관리자가 되어야 합니다:
+
+  예: @내_커뮤니티
+wizard_community_enter_order_channels: |
+  봇에게 구매 및 판매 주문들이 등록될 채널을 하나만 명시하게 된다면, 당신의 커뮤니티 내에서의 모든 주문들은 텔레그램 채널에 등록됩니다. 
+
+  두 개의 채널을 명시할 경우, 구매 주문과 판매 주문은 각기 다른 채널에 등록됩니다. 이 경우, 봇과 당신 모두는 해당 채널들의 관리자가 되어야 합니다.
+
+  채널의 이름을 알려주세요. 두 개의 채널을 원한다면 스페이스로 분리하여 이름들을 명시해주세요.
+  
+  예: @내_판매_커뮤니티 @내_구매_커뮤니티
+wizard_community_one_or_two_channels: 한 개 혹은 두 개의 채널을 명시해야 합니다.
+wizard_community_enter_solvers: 이제, 분쟁 해결을 담당할 사용자들을 스페이스로 분리하여 적어주세요.
+wizard_community_must_enter_names: 적어도 하나의 이름을 추가해야 합니다. 두 개 이상일 경우에는 스페이스로 분리해주세요.
+wizard_community_enter_solvers_channel: 봇이 분쟁 해소 절차 시작 시 알림을 주기 위한 ID나 채널명을 명시해주세요. 채널명 앞에는 @를 붙여주셔야 합니다.
+wizard_community_success: |
+  축하합니다. 당신의 커뮤니티가 생성되었습니다!
+
+  활성화된 커뮤니티를 유지하기 위해서는, 최근 ${days}일 동안 성공적으로 완료된 주문들이 있어야 합니다.
+wizard_community_wrong_permissions: 당신에게는 이 그룹 혹은 채널의 관리자 권한이 없습니다.
+wizard_community_you_are_not_admin: 당신은 ${channel} 채널의 관리자가 아닙니다.
+wizard_community_duplicated_name: 이미 다른 커뮤니티에서 입력한 이름을 사용 중입니다. 다른 이름을 입력해주세요.
+wizard_to_exit: 설정에서 벗어나려면 /exit 명령어를 입력하세요.
+wizard_exit: 설정 모드에서 벗어났습니다. 이제 일반 명령어들을 사용할 수 있습니다.
+wizard_add_fiat_amount: |
+  ${action}하려는 ${currency} 액수를 명시해주세요.
+
+  단순 숫자만을 사용하거나 ${minAmount}와 ${maxAmount}로 이루어진 범위를 제시해야 합니다:
+wizard_add_fiat_wrong_amount: 유효하지 않은 액수입니다. 단순 숫자만을 사용하거나 ${minAmount}와 ${maxAmount}로 이루어진 범위를 제시해야 합니다.
+Invalid amount, input must be between ${minAmount} and ${maxAmount}, it must be only numbers.
```

Comment:
Hi! tests are failing for this line

Reply:
Just updated the line 346. I forgot to remove the original message.